ARImageTrackingExamples/ARImageTrackingExamples/Helpers/Assets.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import UIKit
+
+let assets: [String: String] = [
+    "SOLID-Single":"SOLID-SingleText",
+    "SOLID-Dependency":"SOLID-DependencyText",
+    "SOLID-interface": "SOLID-interfaceText",
+    "SOLID-Liskov": "SOLID-LiskovText",
+    "SOLID-open": "SOLID-openText"
+]
+
+extension String {
```

Comment:
Not sure this extension is really valuable?

Reply:
It made the code cleaner to read than the `UIImage(named: "")`

Reply:
I'm not sure that it is much clearer tbh, just shorter - what it more obvious about what it is doing: `UIImage(named: "foo")` or `"foo".image()`?